queries/luap/highlights.scm

```diff
@@ -32,7 +32,11 @@
 (negated_set
   (character) @constant)
 
-(class) @keyword
+(class) @string.escape
+
+(class
+  "%" @string.regexp
+  (escape_char) @string.regexp)
```

Comment:
Question. Why only % and escape_char are regexp? It should be the reverse

Reply:
Concrete examples always go a long way.

Reply:
Consider the following regex `\w*\.\s\*.` with the equivalent lua pattern `%w*%.%s%*.`

regex | lua pattern | meaning
---   | ---         | ---
`\w`  | `%w`        | character class
`*`   | `*`         | zero or more times
`\.`  | `%.`        | literal dot
`\s`  | `%s`        | character class
`\*`  | `%*`        | literal asterisk
`.`   | `.`         | any character

The regex queries capture character classes as `@string.escape` and escaped special characters (`\.`, `\*`, etc) as `@string.regexp`, but the luap queries captures both as `@keyword`.

Using `(class) @string.escape` will capture character classes. And for escaped special characters: 

```scm
(class
  "%" @string.regexp
  (escape_char) @string.regexp)
```

Reply:
Makes sense to me. @amaanq ?